content/en/tools/digital-currencies/defining/decommissioning-token-definitions.md

```diff
@@ -0,0 +1,99 @@
+---
+date: '2023-03-21'
+lastmod: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: "Decommissioning Token Definitions"
+    weight: 800
+    parent: digital-currencies-token-definitions
+    identifier: digital-currencies-decommissioning-token-definitions
+    description: "Digital Currencies documentation describing how to decommission token definitions via the GUI"
+title: "Decommissioning Token Definitions"
+---
+
```

Comment:
```suggestion
## Decommissioning Token Definitions

```

Reply:
Don't think we need an initial heading as that automatically comes from the Title of the page. Let me know if this is otherwise.